src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -135,11 +136,16 @@ const webhookRouter = async (context: WebhookContext) => {
 			await GithubWebhookMiddleware(codeScanningAlertWebhookHandler)(context);
 			break;
 		case "dependabot_alert":
-			subscriptions = await Subscription.findOneForGitHubInstallationId(context.payload.installation.id, undefined);
+			subscriptions = await Subscription.findOneForGitHubInstallationId(context.payload.installation.id, context.gitHubAppConfig.gitHubAppId);
 			if (subscriptions && await booleanFlag(BooleanFlags.ENABLE_GITHUB_SECURITY_IN_JIRA, subscriptions.jiraHost)) {
 				await GithubWebhookMiddleware(dependabotAlertWebhookHandler)(context);
 			}
 			break;
+		case "secret_scanning_alert":
+			subscriptions = await Subscription.findOneForGitHubInstallationId(context.payload.installation.id, context.gitHubAppConfig.gitHubAppId);
```

Comment:
/nit 
I have a gut feeling that the availability of `subscription` will be checked inside the `GithubWebhookMiddleware`, so not really need to check here. 

I understand we need to jiraHost, but we can put the FF check inside the method of `secretScanningAlertWebhookHandler` where jiraHost is available

Reply:
Good call out. We do check for subscriptions in GithubWebhookMiddleware. I vote we rip this check out (alogn with the one or dependabot_alert.